src/backend/wallpaper.rs

```diff
@@ -0,0 +1,143 @@
+use crate::zvariant::{DeserializeDict, OwnedObjectPath, Type};
+use futures_channel::{
+    mpsc::{Receiver, Sender},
+    oneshot,
+};
+use futures_util::{FutureExt, StreamExt};
+use std::cell::RefCell;
+use zbus::{dbus_interface, names::WellKnownName};
+
+use crate::{backend::IMPL_PATH, desktop::wallpaper::SetOn, WindowIdentifierType};
+
+#[derive(DeserializeDict, Type, Debug, Default)]
```

Comment:
```suggestion
#[derive(DeserializeDict, Type, Debug)]
```
The Default is not needed here